test/BUILD.gn

```diff
@@ -1136,6 +1136,7 @@ test("brave_browser_tests") {
     sources += [
       "//brave/browser/brave_autofill_browsertest.cc",
       "//brave/browser/brave_resources_browsertest.cc",
+      "//brave/browser/renderer_context_menu/test/render_view_context_menu_browsertest.cc",
```

Comment:
you can see that normally we don't create separate "test" directories, so i'd avoid doing that

Reply:
ah, i see the dir already exist. Then it's fine :)